packages/ng/empty-state/empty-state-page/empty-state-page.component.html

```diff
@@ -10,7 +10,7 @@
 		<div class="emptyState-content">
 			<div *ngIf="icon" class="emptyState-content-icon" aria-hidden="true" [innerHtml]="icon | luSafeExternalSvg"></div>
 			<div class="emptyState-content-text">
-				<h1 class="emptyState-content-heading">{{ title }}</h1>
+				<div role="heading" [attr.aria-level]="hx" class="emptyState-content-heading">{{ title }}</div>
```

Comment:
Why not using `<h{{ hx }}>…</h{{ hx }}>`?

Reply:
I don't think that's possible with Angular. It would need to be a custom component that uses `*ngIf` directives to pick the right element.